bundles/org.openhab.core.io.rest.voice/src/main/java/org/eclipse/smarthome/io/rest/voice/internal/VoiceResource.java

```diff
@@ -67,6 +71,7 @@
     UriInfo uriInfo;
 
     private VoiceManager voiceManager;
+    private AudioManager audioManager;
```

Comment:
You should imho do without that dependency - the VoiceManager offers methods to call without sinkIds and it handles the calls internally. The REST API should behave the same way as the Java API.

Reply:
Ok, this was necessary just to make a pre-control and avoid returning status 200 when say will fail because the provided audio sink is wrong.
But I can suppress this test and so this dependency to AudoManager.